src/table/index.ts

```diff
@@ -1 +1,40 @@
-export default {}
+import { uint32Le, assertIsHexStr, littleHexToInt } from '../utils'
+import { deserializeArray } from '../array'
+
+const Uint32Length = 4
+
+export const serializeTable = (origin: Array<TypeElement>): string => {
+  const values = origin.map(item => item[1])
+  values.forEach(item => {
+    assertIsHexStr(item)
+  })
+  values.map(item => (item.length % 2 ? `0${item.slice(2)}` : item.slice(2)))
```

Comment:
Should the result be reassigned to `values`?

Reply:
Updated. https://github.com/Keith-CY/molecule-javascript/pull/8/commits/c4361b5cfa7ac6b3849a29b3dd7c81410ee33a35